static/app/components/quickTrace/index.tsx

```diff
@@ -430,25 +426,30 @@ type DropdownNodeProps = {
   event: TraceError | QuickTraceEvent;
   organization: OrganizationSummary;
   anchor: 'left' | 'right';
+  enableDefault?: boolean;
   onSelect?: (eventKey: any) => void;
+  to?: string | LocationDescriptor;
```

Comment:
Pretty sure just `LocaitonDescriptor` will suffice here because a string path is a valid `LocationDescriptor`.